Documentation/network/servicemesh/istio.rst

```diff
@@ -36,35 +36,45 @@ You can confirm the result with the following command:
     $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep bpf-lb-sock-hostns
     bpf-lb-sock-hostns-only: "true"
 
+
+To ensure that Cilium does not interfere with other CNI plugins on the node, it is important to set the
+``cni-exclusive`` parameter in the Cilium ConfigMap to ``false``. This can be achieved by using the
+``--set`` flag with the ``cni.exclusive`` Helm value set to ``false``.
+You can confirm the result with the following command:
+
+.. code-block:: shell-session
+
+    $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep cni-exclusive
+    cni-exclusive: "false"
+
 .. _gsg_istio_cnp:
 
 Istio configuration
 ===============================
 
-When you deploy Istio, be aware of:
-
-* The new experimental `ambient mesh <https://istio.io/latest/blog/2022/introducing-ambient-mesh/>`_ 
```

Comment:
has ambient mesh been tested with L4 CNPs? If traffic between a pod and a proxy does not egress through a pod's veth, no network policies (CNP or KNP) can be enforced

Reply:
Yes, it has. As of 1.21 all Istio ambient traffic will ingress/egress through the pod normally.

NetPol enforced in the host/node net namespace will apply as you'd expect.

Reply:
is this what's described in https://istio.io/latest/blog/2024/inpod-traffic-redirection-ambient/#istio-ambient-traffic-redirection-the-new-model?

Reply:
Yes.

Reply:
but if I read it correctly, all traffic redirected into a ztunnel proxy will bypass host netNS and, thus, any network policy applied to a pod

Reply:
No, all traffic will be captured once it enters a pod, and before it leaves a pod.

It will always pass through the host netns, and always be subject to CNI NetPol.

Reply:
sorry, I haven't been clear enough. As the traffic leaves the pod, and gets redirected to ztunnel, it will get encrypted, so by the time it hits host netNS it's original src/dst ports are no longer the same, right?

Reply:
Yes, for mTLS traffic leaving the pod, it will be tunneled over port 15008 with an HTTP/2 tunnel.

NetPol would still apply to that traffic just as it would if the pod natively tunneled, but the egress port would change in that case.

This is something we can mention here, or in the Istio docs, or both.

Reply:
+1 on summarizing ^ so a reader understands how the magic happens.

Reply:
I think providing a summary here with links to external resources (esp. around recommendations to disable mTLS) would be enough. We just need to be clear about things that may not work with Ambient

Reply:
@danehans and @networkop PTAL, redid the wording a bit. Hopefully a bit clearer.

Reply:
thanks @bleggett , left a couple of minor comments but overall the wording looks good